tests/kuttl/tests/backup-partial-complete/00-setup.yaml

```diff
@@ -13,7 +13,7 @@ commands:
     namespaced: true
 
   # Patch the inform policies to reflect the compliance status. Make policy 3 compliant with all the clusters.
```

Comment:
Isn't this actually making policy 3 NonCompliant with all the clusters?

Reply:
Updated